.github/workflows/relay.yml

```
@@ -0,0 +1,80 @@
+name: Relay
+
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - "relay/**"
+      - ".github/workflows/relay.yml"
+  pull_request:
+    branches:
+    paths:
+      - "relay/**"
+      - ".github/workflows/relay.yml"
+  workflow_dispatch:
+
+jobs:
+  build-test-publish:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./relay
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+
+      - name: Cache Docker layers
+        uses: actions/cache@v2
+        with:
+          path: /tmp/.buildx-relay-cache
+          key: ${{ runner.os }}-buildx-relay-${{ github.sha }}
+          restore-keys: |
+            ${{ runner.os }}-buildx-relay-
+
+      - run: sudo df -h
+
+      - name: Build Docker Build Image
+        uses: docker/build-push-action@v2
+        with:
+          context: ./relay
+          target: gobuild
+          tags: go-build-env
+          load: true # load image to local registry to use it in next steps
+          cache-from: type=local,src=/tmp/.buildx-relay-cache
+          cache-to: type=local,dest=/tmp/.buildx-relay-cache
+
+      - name: Run Go tests
+        run: |
+          docker run \
+            --workdir /go/src/github.com/keep-network/tbtc/relay \
+            go-build-env \
+            gotestsum
+
+      - name: Login to Google Container Registry
+        if:  |
+          startsWith(github.ref, 'refs/heads/releases/')
```

After [this discussion in Flowdock](https://www.flowdock.com/app/cardforcoin/tech/threads/sqQIWXEcGU8O2Gf-mMqgsfmT6wr) I updated `keep-core` and `keep-ecdsa` workflows to execute steps publishing client only when workflow runs due to merge to master. Changes of the config aren't yet in the master, they're waiting for review in https://github.com/keep-network/keep-core/pull/2378 and https://github.com/keep-network/keep-ecdsa/pull/738. Do we want same rules for publishing of relay maintainer (that wasn't discussed in FD)? If yes, we can apply same config here.
```suggestion
          github.ref == 'refs/heads/master'
            && (github.event_name == 'push'
            || github.event_name == 'workflow_dispatch')
```

---

Yep I think it's a good idea. Please see https://github.com/keep-network/tbtc/commit/724110da01c83b57569db5d7120132bd27732ce0